packages/calcite-components/src/components/action/action.tsx

```diff
@@ -84,6 +88,17 @@ export class Action extends LitElement implements InteractiveComponent {
    */
   @property({ reflect: true }) dragHandle = false;
 
+  /**
+   * When `true` and the `type` is `"expand"`, the component is expanded.
```

Comment:
Could this be a private property for use by Action Bar?

Otherwise - as a user I find this name to be a little confusing - if I'm interpreting the set up correctly - because Action Bar has an `expanded` property itself with different behavior.

It could be the "the component is expanded" verbiage - is this "the component the action corresponds to in the UI is in an expanded state" - MDN uses "toggles a widget" - which we probably want to avoid for .. widget reasons...

Reply:
I don't think it could be internal only. It would have uses outside of action bar. For example any action that toggles content like expand/collapse. The difference is that this will use `aria-expanded` vs `aria-pressed`.